llvm/lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -6288,6 +6288,9 @@ SDValue SelectionDAG::getNode(unsigned Opcode, const SDLoc &DL, EVT VT,
       return getNode(ISD::VECREDUCE_AND, DL, VT, N1);
     break;
   case ISD::SPLAT_VECTOR:
+    // RISC-V vector tuple type is not a vector type.
+    if (VT.isRISCVVectorTuple())
```

Comment:
I have no opinion of this, but this may be too target-specific.

Reply:
Hmm, but I guess there's no way to skip this check lol~